README.md

```diff
@@ -81,7 +82,7 @@ To get a local copy up and running, follow these steps.
 4. Open Terminal.
 5. Change the current working directory to the location where you want the cloned directory.
 6. Type git clone, and then paste the URL you of the reposotory.<br>
-``` $ git clone https://github.com/maximoortelli/Portfolio-setup-and-mobile--first.git ```
+``` $ git clone https://github.com/maximoortelli/HelloMicoverse.git ```
```

Comment:
I suggest as a good practice that you change your current Git Clone URL as this is not the correct repository to clone to. I recommend that you put this " URL: https://github.com/maximoortelli/Portfolio-setup-and-mobile--first.git "👍😃